optics-core/optics-core.cabal

```diff
@@ -97,7 +97,8 @@ library
                    -- internal modules
                    Optics.Internal.Bi
                    Optics.Internal.Fold
-                   Optics.Internal.Indexed
+                   Optics.Internal.Indexed.Classes
+                   Optics.Internal.Indexed.Utils
```

Comment:
I'd keep `Utils` as `Optics.Internal.Indexed`, given the `Optics.Internal.Indexed.Classes` could be some `Data.Traversable.WithIndex` ... later.

Reply:
Fixed :+1: